_posts/2022-10-02-infinispan-14-supporting-duplicates-on-multimap.adoc

```diff
@@ -0,0 +1,63 @@
+---
+layout: blog
+title: Infinispan 14 supporting duplicates on multimap
+permalink: /blog/:year/:month/:day/infinispan-14-supporting-duplicates-on-multimap
+date: '2022-10-2T00:00:00.000-00:00'
+author: miador
+tags: [ "multimap" ]
+---
+
+= Infinispan 14 supporting duplicates on multimap
+
+Dear Infinispan community,
+
+with the Infinispan 14 development release 04 multimap can support duplicates.
```

Comment:
```suggestion
With the Infinispan 14 development release 04 multimap cache supports duplicates.
```
Just a suggestion @miador I would add one more sentence about the benefits that this brings. Why should I use it?